src/style.css

```diff
@@ -1,3 +1,151 @@
+html {
+  .roboto-thin {
+    font-family: Roboto, sans-serif;
```

Comment:
Fix it 
<img width="853" alt="image" src="https://github.com/user-attachments/assets/128cb8be-d82a-4d2c-8629-4dceb73df905">
```suggestion
html {
  font-family: Roboto, Arial, sans-serif;
}
```